README.md

```diff
@@ -194,60 +194,106 @@ populate the following:
+2. Copy the secrets, environmental variables, and drivers from the production
+  setup to the development repository. See [Files not included](#files-not-included).
+3. Build the Docker container targeting the `dev` tag by running
+  `docker compose --profile dev build`.
+4. Start the development environment using `docker compose --profile dev up -d`.
+5. Make your code modifications. Changes in the `src/` directory are reflected
+  in the dev containers due to volume mounts (no need to rebuild).
+6. Submit a job to the development containers using `docker exec`, targeting
+  the development master node (`spark-node-master-dev`).
+7. Check the job status at `$SERVER_IP:8082`, instead of the production port
+  `$SERVER_IP:8080`.
+
+> [!WARNING]
+> The development environment shares the same targets as the production
+> environment. That means it will write to the same S3 bucket/CloudWatch log
+> group and trigger the same workflows/crawlers (though all these features are
+> disabled by default). As such, use this environment carefully. If you mess up
+> production data, you can run the production version of the code to re-fetch it.
+
 ## Scheduling
 
 Batches are currently scheduled via
 [`cron`](https://man7.org/linux/man-pages/man8/cron.8.html). To edit the
 schedule file, use `crontab -e` as the main server user. The example crontab
 file below schedules daily jobs for frequently updated tables and weekly ones
-for rarely-updated tables. Note that the jobs currently _must_ be run as
-user 1003.
+for rarely-updated tables.
 
 ```bash
 # Extract recent years from frequently used tables on weekdays at 1 AM CST
-0 6 * * 1,2,3,4,5 docker exec spark-node-master ./submit.sh --json-string "$(yq -o=json .default_jobs /full/path/to/default_jobs.yaml)"
+0 6 * * 1,2,3,4,5 docker exec spark-node-master-prod ./submit.sh --json-string "$(yq -o=json .default_jobs /full/path/to/default_jobs.yaml)" --run-github-workflow --run-glue-crawler --upload-data --upload-logs
 
 # Extract all tables on Saturday at 1 AM CST
-0 6 * * 6 docker exec spark-node-master ./submit.sh --json-string "$(yq -o=json .weekend_jobs /full/path/to/default_jobs.yaml)"
+0 6 * * 6 docker exec spark-node-master-prod ./submit.sh --json-string "$(yq -o=json .weekend_jobs /full/path/to/default_jobs.yaml)" --run-github-workflow --run-glue-crawler --upload-data --upload-logs
 
 # Extract all test environment tables on Sunday at 1 AM CST
-0 6 * * 7 docker exec spark-node-master ./submit.sh --json-string "$(yq -o=json .weekend_jobs_test /full/path/to/default_jobs.yaml)" --no-run-github-workflow --extract-target test
+0 6 * * 7 docker exec spark-node-master-prod ./submit.sh --json-string "$(yq -o=json .weekend_jobs_test /full/path/to/default_jobs.yaml)" --no-run-github-workflow --run-glue-crawler --upload-data --upload-logs --extract-target test
 ```
 
 ## Structure
 
 Here's a breakdown of important files and the purpose of each one:
 
-```tree
+```bash
```

Comment:
This is just a totally unrelated formatting change I made because I think it's easier to read.